package.json

```
@@ -16,7 +16,8 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "createSampleApp": "./utils/create_sample_app.sh"
```

How about using `yarn create-app` ?

```suggestion
    "create-app": "./utils/create_sample_app.sh"
```

People will use the script to create "actual" apps, and they'll start with the "sample" app. :)